src/styles/index.scss

```diff
@@ -2,2 +1,19 @@
+@use './header.scss';
+@use './card-container';
+
+.page {
+  display: flex;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  scroll-behavior: smooth;
+}
+
+.body {
+  margin: 0;
+  place-content: center;
```

Comment:
The 'place-content: center;' is used to align the content of a flex container. However, the 'display: flex;' is missing in this class.